libs/generic-view/utils/src/lib/with-config/with-config.tsx

```diff
@@ -7,15 +7,28 @@ import React, { ComponentType } from "react"
 import { useSelector } from "react-redux"
 import { ReduxRootState } from "Core/__deprecated__/renderer/store"
 import { RecursiveComponent } from "../models/api-fc.types"
+import { createSelector } from "reselect"
+
+type Keys = { viewKey: string; componentKey: string }
+
+const configSelector = createSelector(
+  (state: ReduxRootState) => state.genericViews.devicesConfiguration,
```

Comment:
you can pass selector as a parameter so if we ever change the field name it will be hidden behind "facade" and only the selector will require an update. same for other similar cases

Reply:
Utils cannot import from `generic-view/store` because there are circular dependencies.